### PR TITLE
Migrate to @liteflow/react library

### PR DESCRIPTION
**Pull request by antho1404**
Update of the starter kit with the new `@liteflow/react` and `@liteflow/core` libraries.
Let's make sure to test all the different flows completely to also ensure the library is working as expected

### CODE REVIEW

File: components/Token/Form/Create.tsx [hunk 1]
```diff
@@ -113,19 +110,27 @@ const TokenFormCreate: FC<Props> = ({
...
-        description: data.description,
-        content: data.content,
-        preview: data.preview,
-        isAnimation: data.isAnimation,
-        isPrivate: data.isPrivate,
-        amount: collection.standard === 'ERC1155' ? parseInt(data.amount) : 1,
-        royalties: parseFloat(data.royalties),
-        traits: [{ type: 'Category', value: data.category }],
-      })
+      debugger
```
Comment (antho1404) [inline components/Token/Form/Create.tsx:113]:
```suggestion
```

File: environment.ts [hunk 2]
```diff
@@ -110,6 +110,15 @@ const environment = {
...
   META_KEYWORDS: 'NFT, marketplace, platform, blockchain, liteflow',
+
+  /**
+   * NFT Mint Behavior
+   */
+  // Enable/disable the lazy minting feature. If enabled, the NFTs will be minted on the first sale
+  LAZYMINT: true,
+
+  // Enable/disable the unlockable content feature. If enabled, the NFTs will have unlockable content only accessible to owners
+  UNLOCKABLE_CONTENT: true,
```
Comment (antho1404) [inline environment.ts:121]: Preserve previous defaults

```suggestion
  LAZYMINT: false,

  // Enable/disable the unlockable content feature. If enabled, the NFTs will have unlockable content only accessible to owners
  UNLOCKABLE_CONTENT: false,
```

File: pages/_app.tsx [hunk 3]
```diff
@@ -222,10 +222,8 @@ function MyApp({ Component, pageProps }: AppProps<MyAppProps>): JSX.Element {
           <CookiesProvider cookies={cookies}>
             <ChakraProvider theme={theme}>
               <LiteflowProvider
-                endpoint={`${
-                  process.env.NEXT_PUBLIC_LITEFLOW_BASE_URL ||
-                  'https://api.liteflow.com'
-                }/${environment.LITEFLOW_API_KEY}/graphql`}
+                apiKey={environment.LITEFLOW_API_KEY}
+                endpoint={process.env.NEXT_PUBLIC_LITEFLOW_BASE_URL}
```
Comment (antho1404) [inline pages/_app.tsx:226]: new interface for the `LiteflowProvider`, `endpoint` is optional but here to ensure that we can test on staging environment or for on-premise installations

File: components/Token/Form/Create.tsx [hunk 4]
```diff
@@ -113,19 +110,26 @@ const TokenFormCreate: FC<Props> = ({
...
+            attributes: [{ traitType: 'Category', value: data.category }],
+            media: {
+              content: data.content,
+              preview: data.preview,
+              isAnimation: data.isAnimation,
+              isPrivate: data.isPrivate,
+            },
+          },
+        },
+        activateLazyMint,
```
Comment (antho1404) [inline components/Token/Form/Create.tsx:131]: lazymint option now passed to the sdk

**Review by ismailToyran — APPROVED** [review]
Working great 🎉 
Tested;
- Updating my seller account
- Creating ERC1155
- Creating ERC721
- Putting up 2 ERC1155 on sale
- Bidding one ERC1155 with my buyer account
- Accepting bid with my seller account
- Cancelling remaining sale (1 in this case) with my seller account
- Putting up ERC721 on auction
- Putting up ERC1155 back to sale with my buyer account
- Buying back ERC1155 from my buyer account via direct sale

I think I've tried all the combinations. Let me know if you want me to test more flows 😉

**Pull request merged**